src/security/jdbc/pom.xml

```
@@ -50,6 +46,12 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>simple-jndi</groupId>
+      <artifactId>simple-jndi</artifactId>
+      <version>0.11.4.1</version>
+      <scope>test</scope>
+    </dependency>
```

Perhaps better to use a maintained fork like https://github.com/h-thurow/Simple-JNDI that has updated dependencies (instead of the version from 2018)

```suggestion
    <dependency>
      <groupId>com.github.h-thurow</groupId>
      <artifactId>simple-jndi</artifactId>
      <version>0.25.0</version>
      <scope>test</scope>
    </dependency>
```

At first glance it seems a drop-in replacement

---

Sounds nice but wonder if it'd break the single-purpose nature of the pr. If others agree I'll update it

---

It's strictly related, I don't see it breaking the single purpose.

---

it's related and not a big deal, but one is upgrading a dependency, and the original purpose is re-enabling a set of tests that are not being run because their setup interferes with the rest of the application in the specific scenario where test dependencies across projects get mixed up. The dependency upgrade has nothing to do with it and I'm afraid of you @aaime and trying to play by the rules :P I'll make the upgrade

---

[done](https://github.com/geoserver/geoserver/compare/46bb91f5311b4ec92e24520bfee0811b83694f34..70ddd8cc4639854fac224036dd158781c849f339), worked out just well:
```
[INFO] Running org.geoserver.security.jdbc.H2JNDIRoleServiceTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.464 s - in org.geoserver.security.jdbc.H2JNDIRoleServiceTest
[INFO] Running org.geoserver.security.jdbc.H2JNDIUserDetailsServiceTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 8.048 s - in org.geoserver.security.jdbc.H2JNDIUserDetailsServiceTest
[INFO] Running org.geoserver.security.jdbc.H2JNDIUserGroupServiceTest
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.184 s - in org.geoserver.security.jdbc.H2JNDIUserGroupServiceTest
```